style.css

```diff
@@ -264,6 +264,113 @@ section {
   list-style: none;
 }
 
+.pop-window {
+  height: 100vh;
+  width: 80%;
+  margin-left: 10%;
+  padding: 2%;
+  background-color: #fff;
+}
```

Comment:
Your design is not following the template design in either version.

Your design:

<img width="184" alt="Screen Shot 2022-06-15 at 8 26 19" src="https://user-images.githubusercontent.com/82426682/173838470-baa47218-86c2-48da-829c-5f18664c8c92.png">

<img width="1440" alt="Screen Shot 2022-06-15 at 8 26 47" src="https://user-images.githubusercontent.com/82426682/173838582-ea544d67-590d-4ded-8bd5-2495582f870f.png">

The template design:

<img width="213" alt="Screen Shot 2022-06-15 at 8 27 28" src="https://user-images.githubusercontent.com/82426682/173838741-95b6a0ad-f9d2-49a1-9e72-d9d9294f9752.png">

<img width="485" alt="Screen Shot 2022-06-15 at 8 27 52" src="https://user-images.githubusercontent.com/82426682/173838831-52206937-b3d3-49cc-bb1d-edbf3025edea.png">

Please fix it.

<img width="952" alt="Screen Shot 2022-06-15 at 8 28 31" src="https://user-images.githubusercontent.com/82426682/173838987-b0efdafa-7a42-4539-a29c-f935f5390f55.png">